iothub/service/src/JobProperties.cs

```diff
@@ -124,13 +124,31 @@ public JobProperties()
         [JsonProperty(PropertyName = "identity", NullValueHandling = NullValueHandling.Ignore)]
         public ManagedIdentity Identity { get; set; }
 
+        /// <summary>
+        /// Whether or not to include configurations in the import or export job.
+        /// </summary>
+        /// <remarks>
+        /// The service assumes this is false, if not specified. If true, then configurations are included in the data export/import.
+        /// </remarks>
+        [JsonProperty(PropertyName = "includeConfigurations", NullValueHandling = NullValueHandling.Ignore)]
+        public bool? IncludeConfigurations { get; set; }
```

Comment:
One has to remember to set this to true, even if they've set the ConfigurationsBlobName property. I accidentally did that and couldn't figure out why my configs weren't being imported.

I've considered setting this to true if/when ConfigurationsBlobName is set. Thoughts?

Reply:
No, while it's an extra step if the customer decides to run this export job again but doesn't want the configurations included it'd potentially overwrite the configs.